torchbenchmark/models/demucs/demucs/augment.py

```diff
@@ -23,7 +23,7 @@ def forward(self, wav):
             if not self.training:
                 wav = wav[..., :length]
             else:
-                offsets = th.randint(self.shift, [batch, sources, 1, 1], device=wav.device)
+                offsets = th.randint(self.shift, [batch, sources, 1, 1], device=wav.device, dtype=th.int64)
```

Comment:
why do we need an additional parameter here?

Reply:
`offsets` is used later as indices for `wav.gather` and operator schema requires it to be int64.  I don't think this is ideal since int can always be upcast to `int64`, but currently we have to make this change.